devtools/maven/src/main/java/io/quarkus/maven/CreateProjectMojo.java

```diff
@@ -77,8 +77,8 @@
     @Parameter(property = "codestartsEnabled", defaultValue = "false")
     private Boolean codestartsEnabled;
 
-    @Parameter(property = "withExampleCode", defaultValue = "true")
-    private Boolean withExampleCode;
+    @Parameter(property = "withoutExampleCode", defaultValue = "false")
+    private Boolean withoutExampleCode;
```

Comment:
Not sure I like renaming this, it's confusing compared to the original setting

Reply:
Since default is with example, IMO it makes more sense to say `withoutExampleCode` when you don't want it.

Reply:
Then IMHO it's more natural to say something like `-DgenerateExample=false`

Reply:
-DemptyExample 
-DnoExample
-DskipExample 

?

Reply:
I think my favorite is skipExample :)

Reply:
:+1:, I like `skipExample` too

Reply:
I'll go with `skipExample` using primitives